community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java

```diff
@@ -615,7 +616,7 @@ public void validate( File[] files )
                             file.getName() + "). Please put such directly on the key, f.ex. " +
                             Options.NODE_DATA.argument() + ":MyLabel" );
                 }
-                Validators.FILE_EXISTS.validate( file );
+                Validators.REGEX_FILE_EXISTS.validate( file );
```

Comment:
Putting regular expressions in filenames like this is a little curious, I think.